testing/path/test_local.py

```diff
@@ -161,7 +161,31 @@ def test_eq_with_strings(self, path1):
         assert path2 != path3
 
     def test_eq_with_none(self, path1):
-        assert path1 != None  # noqa
+        assert path1 != None  # noqa: E711
+
+    @pytest.mark.skipif(
+        sys.platform.startswith("win32"), reason="cannot remove cwd on Windows"
+    )
+    @pytest.mark.skipif(
+        sys.version_info < (3, 0) or sys.version_info >= (3, 5),
+        reason="only with Python 3 before 3.5"
```

Comment:
The reason does not match the check for `sys.version_info >= (3, 5)`.

Reply:
Hmm?  It is only meant to run with py3, but not 3.5+.
So it gets skipped with `< 3` and `>= (3, 5)`.

Reply:
Duh you are right, brain fart on my part 😅